homeassistant/components/netgear/device_tracker.py

```diff
@@ -1,170 +1,134 @@
 """Support for Netgear routers."""
 import logging
-from pprint import pformat
-
-from pynetgear import Netgear
-import voluptuous as vol
-
-from homeassistant.components.device_tracker import (
-    DOMAIN,
-    PLATFORM_SCHEMA,
-    DeviceScanner,
-)
-from homeassistant.const import (
-    CONF_DEVICES,
-    CONF_EXCLUDE,
-    CONF_HOST,
-    CONF_PASSWORD,
-    CONF_PORT,
-    CONF_SSL,
-    CONF_USERNAME,
-)
-import homeassistant.helpers.config_validation as cv
+from typing import Dict
 
-_LOGGER = logging.getLogger(__name__)
-
-CONF_APS = "accesspoints"
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Optional(CONF_HOST, default=""): cv.string,
-        vol.Optional(CONF_SSL, default=False): cv.boolean,
-        vol.Optional(CONF_USERNAME, default=""): cv.string,
-        vol.Required(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_PORT): cv.port,
-        vol.Optional(CONF_DEVICES, default=[]): vol.All(cv.ensure_list, [cv.string]),
-        vol.Optional(CONF_EXCLUDE, default=[]): vol.All(cv.ensure_list, [cv.string]),
-        vol.Optional(CONF_APS, default=[]): vol.All(cv.ensure_list, [cv.string]),
-    }
-)
-
-
-def get_scanner(hass, config):
```

Comment:
Can we start the import step of the config flow here and pass the needed config items to it? Return None after starting the config flow in a task.